saleor/site/migrations/0007_auto_20171027_0856.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.5 on 2017-10-27 13:56
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def link_to_sites(apps, schema_editor):
+    SiteSettings = apps.get_model('site', 'SiteSettings')
+    Site = apps.get_model('sites', 'Site')
+
+    for site in SiteSettings.objects.all():
```

Comment:
`for settings in SiteSettings.objects.all()` - we are iterating over settings objects not sites.